### ISSUE TITLE
[BUG] Discord Rich Presence not working

### ISSUE THREAD

**Issue by HandyHat** [hmlendea/gfn-electron#160]
### Describe the bug

When running GeForce Now, there is no rich presence showing up in Discord.

### To Reproduce

1. Open GeForce Now and Discord
2. Start playing a game
3. Observe how there is no Rich Presence showing up in Discord

### Operating System

Fedora 36

### Desktop Environment

GNOME 42.3

### Display Server

Wayland

### Installation method

flatpak

### Version

1.8.0

### Is this a fresh install of the app or an update from a past version?

Fresh Install

### Did this issue appear right away upon installation/updating, or spontaneously?

Right after installation

### Additional context

Discord is not installed as a snap or flatpak, and other programs do provide a rich presence that works.
I have tried reinstalling GeForce Now.

**Comment by dubfib** [comment]
I works on my end when installing and using the .zip installation
It works both on the homescreen and even when im in the queue/in a game

**Comment by HandyHat** [comment]
The Rich Presence works for me using the .zip installation too, so the issue appears to be with the flatpak/snap versions.

**Comment by hmlendea** [comment]
Does this still happen in the latest version of flatpak?

**Comment by HandyHat** [comment]
Yes. I tested again using v1.11.0 on Fedora 38 and Gnome 44.2. The rich presence works fine when using the .zip installation, but not when using the flatpak version.

**Comment by JusPLP** [comment]
For me, neither the Flatpak nor the zip version is working. Is there a fix for this?

**Comment by hmlendea** [comment]
Many code changes were merged since then, including #297 recently, which addresses DiscordRPC specifically.
Please re-open if this issue still affects the latest version of this app.
